zk/hermez_db/db_xlayer.go

```diff
@@ -59,3 +58,33 @@ func (db *HermezDbReader) GetInnerTxs(blockNum uint64) [][]*types.InnerTx {
 	}
 	return blockInnerTxs
 }
+
+// TruncateInnerTx deletes inner txs from `from` to `to` block number
+func (db *HermezDb) TruncateInnerTx(from, to uint64) error {
+	for i := to; i <= from; i++ {
+		prefix := make([]byte, 8)
+		binary.BigEndian.PutUint64(prefix, i)
+		it, err := db.tx.Prefix(INNER_TX, prefix)
```

Comment:
```suggestion
	for blk := to; blk <= from; blk++ {
		prefix := make([]byte, 8)
		binary.BigEndian.PutUint64(prefix, blk)
		it, err := db.tx.Prefix(INNER_TX, prefix)
```
My opinion, for a person who is new to codebase, we may not understand context of the keys used in innerTxs. This could help make it easier to understand that inner txs are grouped together under block number.